aws-crt-kotlin/native/src/aws/sdk/kotlin/crt/util/DigestNative.kt

```diff
@@ -4,12 +4,13 @@
  */
 package aws.sdk.kotlin.crt.util
 
+import aws.sdk.kotlin.crt.WithCrt
 import aws.sdk.kotlin.crt.util.hashing.Sha256
 
 /**
  * Utility object for various hash functions
  */
-public actual object Digest {
+public actual object Digest : WithCrt() {
```

Comment:
Why does this need `WithCrt()`? It uses `Sha256` but that already has `WithCrt` applied: https://github.com/awslabs/aws-crt-kotlin/blob/d147cce9d2195e65a56143d0d8e4d45280c0994e/aws-crt-kotlin/native/src/aws/sdk/kotlin/crt/util/hashing/ShaNative.kt